app/.env.example

```diff
@@ -0,0 +1,58 @@
+#############
```

Comment:
If we want to go ahead with the removal of the `env` file, we should remove this file